packages/core/test/actions.test.ts

```diff
@@ -1765,29 +1763,36 @@ describe('sendTo', () => {
         waiting: {
           on: {
             EVENT: {
-              guard: (_, e) => e.count === 42,
               actions: () => done()
             }
           }
         }
       }
     });
 
-    const parentMachine = createMachine<{
-      child: ActorRefFrom<typeof childMachine>;
-      count: number;
-    }>({
-      context: () => ({
-        child: spawn(createMachineBehavior(childMachine)),
-        count: 42
-      }),
-      entry: sendTo(
-        (ctx) => ctx.child as any,
-        (ctx) => ({ type: 'EVENT', count: ctx.count })
-      )
+    const parentMachine = createMachine({
+      context: ({ spawn }) => {
+        return {
+          child: spawn(createMachineBehavior(childMachine), 'child')
+        } as {
+          child: ActorRefFrom<typeof childMachine>;
+          count: number;
+        };
+      },
+      entry: sendTo((ctx) => ctx.child, { type: 'EVENT' })
     });
 
-    interpret(parentMachine).start();
+    interpret(parentMachine)
+      .onTransition((state) => {
+        state.actions.forEach((action) => {
+          if (action.type === 'xstate.invoke') {
+            expect((action as InvokeActionObject).params.ref).toBe(
+              state.children['child']
+            );
+          }
+        });
+      })
+      .start();
```

Comment:
I'm not sure how this assertion related to the intention of this test

```suggestion
    interpret(parentMachine).start();
```

Reply:
👋